fpr/templates/fpr/format/list.html

```diff
@@ -38,12 +38,12 @@
         <tbody>
         {% for format in formats %}
           <tr>
-            <td><a href="{% url 'format_detail' format.slug %}">{{ format.description }}</a></td>
+            <td><a href="{% url 'format_detail' format.slug %}">{{ format.description }} ({{ format.get_pronom_ids }})</a></td>
             <td>
               {% if request.user.is_superuser %}
-                <a href="{% url 'formatgroup_edit' format.group.slug %}">{{ format.group }}</a>
+                <a href="{% url 'formatgroup_edit' format.group.slug %}">{{ format.group.slug }}</a>
               {% else %}
-                {{ format.group }}
+                {{ format.group.slug }}
```

Comment:
You mention in the description that this is fixing a bug - what is the bug? The list was working fine before this change.

Reply:
Interesting.. I wasn't seeing any groups before this!